sql/item_mods.sql

```diff
@@ -208,10 +208,14 @@ INSERT INTO `item_mods` VALUES (10293,1,2); -- Chocobo Shirt - DEF +2
 INSERT INTO `item_mods` VALUES (10295,8,4); -- Kokou's Earring - STR +4
 INSERT INTO `item_mods` VALUES (10295,25,4); -- ACC +4
 INSERT INTO `item_mods` VALUES (10295,73,1); -- STP +1
+-- INSERT INTO `item_mods` VALUES (10296,AUTOMATON:ACC,3); -- CHARIVARI EARRING, Automaton:Acc+3, RACC+3, MACC+3
+-- INSERT INTO `item_mods` VALUES (10296,AUTOMATON:RACC,3);
+-- INSERT INTO `item_mods` VALUES (10296,AUTOMATON:MACC,3);
 INSERT INTO `item_mods` VALUES (10297,27,-2); -- Sortiarius Earring - Enmity -2
 INSERT INTO `item_mods` VALUES (10297,28,6); -- MAB +6
 INSERT INTO `item_mods` VALUES (10298,111,4); -- Beatific Earring - DIV SKILL +4
 INSERT INTO `item_mods` VALUES (10298,112,4); -- HEAL SKILL +4
+-- INSERT INTO `item_mods` VALUES (10299,PET:DATK%,2); -- SABONG EARRING, Pet:Double Atk+2%
```

Comment:
item_mods_pet table for this. Do note that it doesn't currently have a way to differentiate between types of pets, so I'd only add anything that affects all pets like this one.